src/components/vm/vmActions.tsx

```diff
@@ -207,11 +207,11 @@ export async function addTPM(vm: VM): Promise<void> {
     await virtXmlAdd(vm, "tpm", "default");
 }
 
-const onAddTPM = async (vm: VM, onAddErrorNotification: (n: Notification) => void) => {
+const onAddTPM = async (vm: VM) => {
     try {
         await addTPM(vm);
     } catch (ex) {
-        onAddErrorNotification({
+        addNotification({
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2371-dc83a907-20251022-115647-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)